app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala

```diff
@@ -99,6 +99,17 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     )
   }
 
+  def auditCaseReviewChange(oldCase: Case, updatedCase: Case, operator: Operator)
+                           (implicit hc: HeaderCarrier): Unit = {
+    sendExplicitAuditEvent(
+      auditEventType = CaseReviewChange,
+      auditPayload = baseAuditPayload(updatedCase, operator) + (
+        "newReviewStatus" -> updatedCase.decision.flatMap(_.review).map(_.status.toString).getOrElse("None"),
```

Comment:
**DRY principle**

create private method called `reviewStatus` where given a case `c`, you extract its review status:
```
c.decision.flatMap(_.review).map(_.status.toString).getOrElse("None")
```
Then you can call the method twice, avoiding the same code to be written twice

Reply:
see what I have done for the appeal status